release-engineering/role-handbooks/patch-release-team.md

```diff
@@ -256,6 +263,174 @@ For each cherry-pick request:
+     additional external context not yet publicly in the GitHub PR.
+
+ * ***Submitter*** - human contact for the cherry pick PR
+
+ * ***SIG*** - comma separated list of SIGs labelled on the cherry pick PR
+
+ * ***Notes*** - short, human readable summary of the cherry pick PR, eg:
+
+   * "race in the ECR credentials provider kubernetes -> kubelet panic"
+
+   * "NPE in apiserver(?) and/or cli{?)"
+
+   * "upgrade release-1.13 to go1.11.13"
+
+   * In general our community does not provide the best of PR summary lines
+     and it can take considerable time for each patch release team member to
+     get to an understanding of the PR content.  Once one of us has done
+     that, storing it helps the rest of us as we collaboratively assess the
+     merit of the cherry pick.
+
+ * ***Disposition*** - "to merge?", "to close?", "to merge", "merging", "merged", "hold", "closed"
+
+   * current short form thinking of the team regarding where the cherry pick is
+
+   * "merging": when applying the ```cherry-pick-approved``` label in GitHub, update
+     the disposition to "merging" and highlight it bright yellow.  These PRs
+     will fall off the standard query of ```cherry-pick-not-approved``` PRs,
+     but may not actually merge for example due to a test failure or a Tide
+     hiccup, yet they need tracked until they have actually fully merged.
+     Once merged the cell can move back to a basic green.
+
+ * ***TODO*** - multi-line additional details on status and actions, where each
+  line may be, eg:
+
+   * "needs lgtm"
+
+   * "needs approve"
+
+   * "needs lgtm/approve"
+
+   * "CVE-2018-1002105"
+
+   * For many of the cherry picks, there is a corresponding cherry pick on
+     other release branches which should be noted, eg:
+
+     ```
+     CP in other releases:
+     1.13 https://github.com/kubernetes/kubernetes/pull/82503
+     1.14 https://github.com/kubernetes/kubernetes/pull/82502
+     1.15 https://github.com/kubernetes/kubernetes/pull/82384
+     ```
+
+     Filling this out means that once an issue is triaged for one branch,
+     it's just a copy/paste of that row into the spreadsheet for the other
+     branches.
+
+   * It can be useful to note your name and a date next to a line of text
+     here, especially early during the triage where we're trying to come to
+     a consensus on the merit of the cherry pick.
+
+A number of the fields here come from the parent PR automatically
+when the cherry pick automation is used (this should be the norm
+for most PRs).  The parent PRs may have poor labelling resulting
+in incomplete labels on the cherry pick.  Attempt to clean then up
+if possible or comment asking for owners to better disposition them.
+When recording unsure status from labels add a "?" in the spreadsheet
+cell to indicate the uncertainty.
+
+### Future Work
+
+ * It would be awesome to replace this manual stateful review workflow with a
+   tool.  Such a tool appears to need invented.
```

Comment:
what are some features or specs for such a tool? if it's needed very much, let's start to think about it.

Reply:
No spec exists, but automation in support of the manual inspect, triage, share thoughts, discuss, form consensus would be useful.  As I write that...It almost sounds like Gerrit.  /me ducks from the thrown tomatoes.